maps/MODULES/binding.lua

```diff
@@ -80,10 +127,18 @@ battleStack_mt = {
 			return GetStackSide(t)
 		elseif k == "type" then
 			return GetStackType(t)
+		elseif k == "creatureIdx" then
```

Comment:
I used creatureIdx because this was the actual name of the property, but I will rename it as "creatureType"